### PR TITLE
feat: enable alerts and reports

### PR DESCRIPTION
**Pull request by Ian2012**
This PR enables alerts and reports

Fixes: #310

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutoraspects/templates/aspects/build/aspects-superset/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 authlib  # OAuth2
 clickhouse-connect>0.5,<0.6
 sentry-sdk[flask]
+urllib3>=1.26.15,<2
```
Comment (Ian2012) [inline tutoraspects/templates/aspects/build/aspects-superset/requirements.txt:4]: `urlib3==2.0.0` introduced a change that broke `selenium`. For this reason is pinned

File: tutoraspects/templates/aspects/build/aspects-superset/Dockerfile
Comment (Ian2012) [inline tutoraspects/templates/aspects/build/aspects-superset/Dockerfile:1]: Extracted from https://github.com/apache/superset/blob/2.1.0/Dockerfile#L109-L141

**Comment by bmtcril** [comment]
Seems ok to me aside from the one comment, is it all working now?

File: tutoraspects/templates/aspects/apps/superset/pythonpath/superset_config.py [hunk 2]
```diff
@@ -123,11 +123,23 @@ class CeleryConfig(object):
...
 
+# Email configuration
+SMTP_HOST = "{{SMTP_HOST}}" # change to your host
+SMTP_PORT = {{SMTP_PORT}} # your port, e.g. 587
+SMTP_STARTTLS = {{SMTP_USE_TLS}}
+SMTP_SSL = {{SMTP_USE_SSL}}
+SMTP_USER = "{{SMTP_USERNAME}}" # use the empty string "" if using an unauthenticated SMTP server
+SMTP_PASSWORD = "{{SMTP_PASSWORD}}" # use the empty string "" if using an unauthenticated SMTP server
+SMTP_MAIL_FROM = "{{CONTACT_EMAIL}}"
+EMAIL_REPORTS_SUBJECT_PREFIX = "[Superset] "
```
[the remark this answers is not in the document]
Comment (Ian2012) [reply tutoraspects/templates/aspects/apps/superset/pythonpath/superset_config.py]: Updated

**Review by bmtcril — APPROVED** [review]
👍

**Pull request merged**